### ISSUE TITLE
Is the convert-adjust setting broken?

### ISSUE THREAD

**Issue by deepfakesclub** [deepfakes/faceswap-playground#121]
I always use the "Masked" setting, but today I tried the "Adjust" setting under convert. The resulting merged faces are wrong - it looks like the converted face is not rescaled to the correct size? I tested an older commit (pre-keras/pytorch) and had the same result.

Has anyone else noticed this, or am I doing something wrong?

**Comment by bryanlyon** [comment]
Can you post some example images (preferably cropped around the faces plus a bit)?

**Comment by deepfakesclub** [comment]
![bb23a1](https://user-images.githubusercontent.com/36589430/38357034-12a17df8-38f4-11e8-894f-b0f31287c1a6.jpg)
![bb23a1](https://user-images.githubusercontent.com/36589430/38357048-199cbca8-38f4-11e8-92b7-ad18c2dde220.jpg)

The first is with Masked (Original model all) the second is with Adjust. I didn't bother training the model ore than a few minutes from scratch but you can see what I mean.

I'm simply asking if anyone has been using "Adjust" successfully as it appears broken in my hands for a long time or I am using it incorrectly.

**Comment by ruah1984** [comment]
@deepfakesclub , do you still found the issue?? i found the same problem before, the image not align correctly to the face. I was running GAN 128 that time, i thought is only happen in GAN version, but seems like the "adjust" setting having the problem.
